fenix/app/metrics.yaml

```diff
@@ -9538,6 +9657,23 @@ credit_cards:
         - Autofill
 
 addresses:
+  autofill:
```

Comment:
I don't see anything using this.

Reply:
ah turns out we already had `addresses.autofilled` which would measure the same thing, removed.  thanks for catching this!